### ISSUE TITLE
Feat: want to add a popup for confirmation 

### ISSUE THREAD

**Issue by Shreyaa173** [Sanchitbajaj02/palettegram#406]
### What feature?

this will confirm that data has been saved

### Add screenshots

![image](https://github.com/user-attachments/assets/5ff2600d-2fe8-4683-9669-efa125e7d87b)

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I want to work on this issue

**Comment by Sanchitbajaj02** [comment]
@Shreyaa173 Please explain what type of popup?

**Comment by Shreyaa173** [comment]
it is kindoff a confirmation popup that the user has subscribed

**Comment by ishita-43** [comment]
Pop like when sites give you successfully signed in ?? Or you will design this some other way

**Comment by Shreyaa173** [comment]
Nope, it would be like you have successfully subscribed to our newsletter. 
![image](https://github.com/user-attachments/assets/ee436c5f-dfde-4b52-9d4f-15eeadd1a181)
something like this

**Comment by ishita-43** [comment]
Can  u design the element according to palettegram theme
And will this take into account the confirmarion of email sent to the user? What added features will this add to the project

**Comment by Sanchitbajaj02** [comment]
> Can u design the element according to palettegram theme And will this take into account the confirmarion of email sent to the user? What added features will this add to the project

Hi @Shreyaa173 assigning the issue to you. Make sure you follow these guidelines, then only it gets merged
